var/spack/repos/builtin/packages/salmon/package.py

```diff
@@ -85,6 +85,13 @@ class Salmon(CMakePackage):
             when="@{0}".format(ver),
         )
 
+    patch(
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
Done 👍